testgrid/config/config.yaml

```diff
@@ -1806,12 +1806,16 @@ dashboards:
   dashboard_tab:
   - name: gke-cvm-1.6-cvm-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-cvm-stable-cvm-master-upgrade-master
+    description: 'Upgrade master only, in gke, from container_vm 1.6 to container_vm 1.8'
   - name: gke-cvm-1.6-gci-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-cvm-stable-gci-master-upgrade-master
+    description: 'Upgrade master only, in gke, from container_vm 1.6 to gci 1.8'
   - name: gke-gci-1.6-cvm-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-gci-stable-cvm-master-upgrade-master
+    description: 'Upgrade master only, in gke, from gci 1.6 to container_vm 1.8'
   - name: gke-gci-1.6-gci-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-gci-stable-gci-master-upgrade-master
+    description: 'Upgrade master only, in gke, from gci 1.6 to gci 1.8'
   - name: gce-1.7-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gce-new-master-upgrade-master
```

Comment:
Would that change if we later modify the gce e2e scripts? It would be hard to keep them in sync. Probably just don't mention it?